tests/gdx-tests/src/com/badlogic/gdx/tests/math/OctreeTest.java

```diff
@@ -0,0 +1,178 @@
+/*******************************************************************************

```

Comment:
i think we need a real example : 
* Octree<GameObject> where GameObject class holds a model instance and a cached boundary box.
* Query with frustrum to know which object to render (could be a faked camera to see how octree work ?)
* RayCast on click that could highlight our game object.
* Displaying internal AABB should be enabled/disabled via a toggle button or a keyboard key.

Reply:
Done!
![Screenshot from 2020-11-11 22-47-59](https://user-images.githubusercontent.com/1824878/98885058-1d36cd80-2470-11eb-8cc1-4f439af27234.png)